app/Http/Controllers/PostController.php

```diff
@@ -53,14 +53,21 @@ public function store(Request $request)
     // 個別の投稿を表示するが疑似SPAで事実上タイムライン
     public function show(string $id)
     {
-        $post = Post::findOrFail($id);
+        $post = Post::with(['user', 'music'])->findOrFail($id);
         return view('posts.show', compact('post'));
     }
 
     // タイムライン用のエンドポイント(json)
     public function load_more()
```

Comment:
nitpick: 型定義

直さなくてもいいんだけど、他の人のレビューで指摘してるから参考までに。
型定義をすると、型安全になるしIDEに情報を渡せるから積極的にやるといいよ！

```suggestion
use Illuminate\Http\JsonResponse;
    
　  /**
     * @return JsonResponse
     */
    // タイムライン用のエンドポイント(json)
    public function load_more(): JsonResponse

```